lib/objectid.js

```diff
@@ -0,0 +1,12 @@
+var ObjectId;
+
+try {
+  ObjectId = require('mongodb').ObjectId;
+} catch(e) {
+  if (e.code === 'MODULE_NOT_FOUND')
+    throw e;
```

Comment:
Fair enough. Please add a debug log to let the user know `ObjectId` will be undefined because `mongodb` was not found.

```js
debug('Cannot load mongodb, ObjectId will be undefined. %s', e.stack);
```